### PR TITLE
General work on releases

### LINKED ISSUE #736: [RELEASE] mc  RELEASE.2023-02-16T19-20-11Z
`mc` [RELEASE.2023-02-16T19-20-11Z](https://github.com/minio/mc/releases/tag/RELEASE.2023-02-16T19-20-11Z) has updates that impact the docs.

- [x] `mc rm` no longer allows the `--purge` command with any flag other than `--force` [PR #4467](https://github.com/minio/mc/pull/4467)
- [x] `mc admin bucket remote bandwidth` moved into `mc replicate status` command [PR #4430](https://github.com/minio/mc/pull/4430)
- [ ] new `mc admin user sts info` command [PR #4377](https://github.com/minio/mc/pull/4377)

### LINKED ISSUE #750: [Release] Console v0.25.0 doc impacts
Console [v0.25.0](https://github.com/minio/console/releases/tag/v0.25.0) updates the login screen.

- [ ] Review the `administration/minio-console/#logging-in` page for updates to the flow. ([PR #2695](https://github.com/minio/console/pull/2695))

### PR DESCRIPTION
**Pull request by ravindk89**
Closes #750 
Closes #736 
Partially Address #767 767

I noticed that for `mc admin config get`, `enable` is like a global flag that everyone gets. The entire page needs restructuring so that is something to tackle down the line.

### CODE REVIEW

File: source/reference/minio-mc-admin/mc-admin-config.rst [hunk 1]
```diff
@@ -1984,6 +1984,7 @@ configuration settings.
       :class: copyable
 
       mc admin config set identity_ldap \
+         enabled="true"
```
Comment (djwfyi) [inline source/reference/minio-mc-admin/mc-admin-config.rst:1987]: Need the line ending `\` or we'll get yelled at.

File: source/reference/minio-mc-admin/mc-admin-config.rst [hunk 2]
```diff
@@ -1996,26 +1997,14 @@ configuration settings.
...
 
       .. include:: /includes/common-minio-external-auth.rst
-         :start-after: start-minio-ad-ldap-sts-expiry
-         :end-before: end-minio-ad-ldap-sts-expiry
+         :start-after: start-minio-ad-ldap-server-addr
+         :end-before: end-minio-ad-ldap-server-addr
 
       This environment configuration setting with the 
-      :envvar:`MINIO_IDENTITY_LDAP_STS_EXPIRY` environment variable.
+      :envvar:`MINIO_IDENTITY_LDAP_SERVER_ADDR` environment variable.
```
Comment (djwfyi) [inline source/reference/minio-mc-admin/mc-admin-config.rst:2007]: This isn't a complete sentence.
It's all over a lot of the existing page, though. But it's missing something.

Is it supposed to be something like:
> Alternatively, configure this option with the .... environment variable.

File: source/reference/minio-mc-admin/mc-admin-config.rst [hunk 3]
```diff
@@ -2065,6 +2054,29 @@ configuration settings.
...
+      :delimiter: " "
+
+      *Optional*
+
+      .. include:: /includes/common-minio-external-auth.rst
+         :start-after: start-minio-ad-ldap-sts-expiry
+         :end-before: end-minio-ad-ldap-sts-expiry
+
+      This environment configuration setting with the 
+      :envvar:`MINIO_IDENTITY_LDAP_STS_EXPIRY` environment variable.
```
Comment (djwfyi) [inline source/reference/minio-mc-admin/mc-admin-config.rst:2078]: Again, this isn't a complete sentence.

File: source/reference/minio-mc-admin/mc-admin-config.rst [hunk 4]
```diff
@@ -2065,6 +2054,29 @@ configuration settings.
...
       :envvar:`MINIO_IDENTITY_LDAP_USER_DN_SEARCH_FILTER` environment variable.
 
+   .. mc-conf:: enabled
+      :delimiter: " "
+
+      *Optional*
+
+      Set to ``false`` to disable the AD/LDAP configuration.
+
+      Applications cannot generate STS credentials or otherwise authenticate to MinIO using the configured provider if set to ``false``.
```
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-config.rst:2064]: Maybe reverse this sentence to put `if set to false` at the beginning. This is about the setting first, then what applications may or may not do as a result. (Also in the OpenID section below.)

**Review by djwfyi — CHANGES_REQUESTED** [review]
Minor nit.
But also, the config page seems to have a bunch of incomplete thoughts/sentences all over it, two of them appearing in the diff here. But the same sentence fragment is all over the page.

File: source/reference/minio-mc-admin/mc-admin-config.rst (hunk 4 above)
Comment (djwfyi) [reply source/reference/minio-mc-admin/mc-admin-config.rst:2064]: Good catch and suggestion. I concur.

File: source/reference/minio-mc-admin/mc-admin-config.rst (hunk 2 above)
Comment (djwfyi) [reply source/reference/minio-mc-admin/mc-admin-config.rst:2007]: Oh, wait maybe:

> This configuration option is the same as setting the ...

(And maybe noting that env var override config settings?)

Comment (ravindk89) [reply source/reference/minio-mc-admin/mc-admin-config.rst:2007]: There's a "corresponds with" missing in here :grimacing:

File: source/reference/minio-mc-admin/mc-admin-config.rst (hunk 4 above)
Comment (ravindk89) [reply source/reference/minio-mc-admin/mc-admin-config.rst:2064]: Oooh good suggestion, taken.

File: source/reference/minio-mc-admin/mc-admin-config.rst [hunk 5]
```diff
@@ -1996,26 +1997,14 @@ configuration settings.
...
-      :end-before: end-minio-ad-ldap-server-addr
-
-   This environment configuration setting with the 
-   :envvar:`MINIO_IDENTITY_LDAP_SERVER_ADDR` environment variable.
-
-   .. mc-conf:: sts_expiry
-      :delimiter: " "
-
-      *Optional*
+      *Required*
```
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-config.rst:2000]: Do `:optional:` and `:required:` work in this context? Or is the manual style the only way.

File: source/reference/minio-mc-admin/mc-admin-config.rst (hunk 3 above)
Comment (ravindk89) [reply source/reference/minio-mc-admin/mc-admin-config.rst:2078]: Fixed throughout. Along with a few other whoopsies. Thankful for find replace.

**Review by ravindk89 — COMMENTED** [review]
@djwfyi @feorlen back to you both!

**Review by feorlen — APPROVED** [review]
LGTM

**Pull request merged**